core/server/master/src/test/java/alluxio/master/AlluxioMasterProcessTest.java

```diff
@@ -171,32 +171,35 @@ public void restoreFromBackupLocal() throws Exception {
 
   @Test
   public void startZeroParallelism() {
-    ServerConfiguration.set(PropertyKey.MASTER_RPC_EXECUTOR_PARALLELISM, "0");
+    ServerConfiguration.set(PropertyKey.MASTER_RPC_EXECUTOR_TYPE, "FJP");
```

Comment:
can we also add some sanity test for TPE?

Reply:
Al those that are asserted are tested already. (keepalive etc is effective for both pools)